linkerd/app/src/env.rs

```diff
@@ -280,6 +283,13 @@ pub fn parse_config<S: Strings>(strings: &S) -> Result<super::Config, EnvError>
 
     let hostname = strings.get(ENV_HOSTNAME);
 
+    let labels = strings.get(ENV_LABELS_FILE_PATH).map(|path_option| {
```

Comment:
Would you mind either: (1) renaming `labels` to `oc_trace_labels` or (2) moving this into a function and invoking at lines 462...

"Labels" means several things in the context of the proxy, so I want to be very clear about use of this term here... In this case, these labels are only used for tracing exports (which is not used the majority of the time, so it seems generally preferable to only execute this when tracing is enabled).